docs/integrations/sources/netsuite-odbc.md

```diff
@@ -0,0 +1,165 @@
+
+#### Step 2.4: Setup Your Suite Analytics Role
+
+1. Go to **Setup** » **Users/Roles** » **Manage Roles** » **New**
+2. Fill in the **Name** field (we recommend using something memorable like 'Airbyte OBDC Role')
+3. Scroll down to **Permissions** tab
+4. Click on `Setup` and manually add the `Log In Using Access Tokens` and `SuiteAnalytics Connect` permissions
+5. Click Save
+
+#### Step 2.5: Setup User
+
+1. Go to **Setup** » **Users/Roles** » **Manage Users**
+2. Decide which user you want to give permissions to
+3. In column `Name` click on the user’s name
+4. Then click on **Edit** button under the user’s name
+5. Scroll down to **Access** tab at the bottom
+6. Make sure the `Give Access` checkbox is enabled
+7. In the **Roles** section select the `Data Warehouse Integrator Role`
+8. In the **Roles** section select the dedicated SuiteAnalytics role you created in step 2.4
+9. Click Save
+
+#### Step 2.6: Create Access Token for role
+
+1. Go to **Setup** » **Users/Roles** » **Access Tokens** » **New**
+2. In the **Application Name** input, select the Integration Record you created in step 2.3
+3. Under **User** select the user you assigned roles to in step **2.5**
+4. Inside **Role** select the Data Warehouse Integrator Role
+5. Under **Token Name**, you should provide a descriptive name to the Token you are creating
+6. Save changes
+7. After that, **Token ID** and **Token Secret** will be showed at the bottom of the page. Copy these and save them somewhere safe. **They will only be shown once.**
+
+8. Get Data Warehouse integrator Role ID
+   1. Set up -> Users/Roles -> Manage Roles
+   2. Find the internal ID of the Data Warehouse Integrator role
+
+#### Step 2.7: Get the Data Warehouse Integrator Role ID
+
+1. Go to **Setup** » **Users/Roles** » **Manage Roles**
+2. Find the internal ID of the Data Warehouse Integrator role. Copy and save it
+
+### Step 3: Make Sure You Have the required Information
+
+Before moving forward, you should check to see that you have:
+
+- Service Host
+- Service Port
+- Account ID
+- Consumer ID
+- Consumer Secret
+- Token ID
+- Token Secret
+- Date Warehouse Integrator Role ID
+
+You should also have a correctly configured User with permissions to access Suite Analytics Connect
+
+### Step 4: Set up the source connector in Airbyte
+
+### For Airbyte Cloud:
+
+1. [Log into your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
+2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ new source**.
+3. On the source setup page, select **Netsuite ODBC** from the Source type dropdown and enter a name for this connector.
+4. Add **Account ID**
+5. Add **Consumer Key**
+6. Add **Consumer Secret**
+7. Add **Token ID**
+8. Add **Token Secret**
+9. Add **Service Host**
+10. Add **Service Port**
+11. Add **Role ID**
```

Comment:
which role is this supposed to be? the data integration warehouse one or the one created in step 2.4?

Reply:
I assume, based on the steps, that this is the integration DWH as we are telling them to get it and save it. But, Do you think it is obvious or necessary to have a clarification note This is the DWH integrator Role ID? just to be safe?